air-quality-backend/src/air_quality/etl/in_situ/openaq_dao.py

```diff
@@ -25,11 +25,11 @@ def call_openaq_api(city, date_from: datetime, date_to: datetime) -> list:
         "sort": "desc",
         "order_by": "datetime",
         "radius": "25000",
-        "date_to": date_to.strftime("%Y-%m-%dT%H:%M:%S"),
-        "date_from": date_from.strftime("%Y-%m-%dT%H:%M:%S"),
+        "date_to": date_to.strftime("%Y-%m-%dT%H:%M:%S%z"),
+        "date_from": date_from.strftime("%Y-%m-%dT%H:%M:%S%z"),
         "coordinates": format_coordinates(city),
         "parameter": ["o3", "no2", "pm10", "so2", "pm25"],
-        "unit": "µg/m³",
+        "unit": ["µg/m³", "ppm"],
```

Comment:
Do you think we should remove this filter entirely? I remember it noticeably improves the speed of the response, but also we could see if anything else ever comes back (maybe filter them out and log?)

Reply:
Good idea, have done